library/src/main/kotlin/build/buf/connect/protocols/ConnectInterceptor.kt

```diff
@@ -46,9 +46,7 @@ import java.net.URL
 internal class ConnectInterceptor(
     private val clientConfig: ProtocolClientConfig
 ) : Interceptor {
-    private val moshi = Moshi.Builder()
-        .add(KotlinJsonAdapterFactory())
```

Comment:
Moshi supports either reflection based serialization or serialization using codegen. This removes the factory which powers reflection support.